DocsXsd/phpcsdocs.xsd

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema" elementFormDefault="qualified">
+    <xs:element name="documentation">
+        <xs:complexType>
+            <xs:sequence maxOccurs="unbounded">
+                <xs:group ref="rulegroup"/>
+            </xs:sequence>
+            <xs:attribute name="title" use="required" type="titleType"/>
+            <xs:anyAttribute processContents="lax"/>
+        </xs:complexType>
+    </xs:element>
+
+    <xs:group name="rulegroup">
+        <xs:sequence>
+            <xs:element name="standard" type="standardType"/>
+            <xs:element name="code_comparison" type="code_comparisonType" maxOccurs="unbounded"/>
```

Comment:
Should/Can this have a `minOccurs="0"` ? See also my more extensive comment regarding the same with the `InvalidMissingCodeComparisonElement` test.